### ISSUE TITLE
Login form is not interactive

### ISSUE THREAD

**Issue by Che140401** [DSC-JSS-NOIDA/HackHive#2]
Login form of this website looks very weired please assign it to me im front-end developer 
Name:Chetan Karande
Work Experience:front End developer
![login form](https://github.com/DSC-JSS-NOIDA/HackHive/assets/109230049/3a48c6ed-d1d2-41c4-969a-83904da2d880)

**Comment by aswathcm29** [comment]
Yeah I'm really interested in doing the login form interactive to you ! Please do assign me to this issue and so I'll complete this asap

**Comment by RamitVishwakarma** [comment]
Assigned the issue to both of you 
Make sure to look at the design file before starting work on this
here's the design file https://www.figma.com/design/9HzyqaEl4SXVY4iZIA0SFp/HackHive

**Comment by Che140401** [comment]
@RamitVishwakarma 
**Issue regarding to run this project !!!!!**

When i run as command npm run dev it is showing me blank page regarding this how to run frontend folder ?? should we use another command

**Comment by RamitVishwakarma** [comment]
> @RamitVishwakarma 
> **Issue regarding to run this project !!!!!**
> 
> When i run as command npm run dev it is showing me blank page regarding this how to run frontend folder ?? should we use another command
> 

Have you changed directory to the frontend folder?
do cd frontend before running npm i and npm run dev

**Comment by TomLucasakaTGeek** [comment]
I would like to work on this issue, please assign this to me.

**Comment by tejaswini4996** [comment]
Will  you please assign it to me?
